internal/pkg/log/hook_filter.go

```diff
@@ -50,5 +51,13 @@ func (f *FilterHook) Fire(entry *logrus.Entry) error {
 	if entry.Level == logrus.DebugLevel {
 		entry.Message = f.predefined.ReplaceAllString(entry.Message, "[FILTERED]")
 	}
+
+	// Uppercase every first character for each log just for better optics
+	if len(entry.Message) > 0 {
+		messageBytes := []byte(entry.Message)
+		messageBytes[0] = byte(unicode.ToUpper(rune(messageBytes[0])))
+		entry.Message = string(messageBytes)
+	}
+
```

Comment:
Neat!  Now if you can manage to put a period at the end of each line!   
Just kidding on that pat.

Reply:
:grin:  We could also pipe it though Google translate and provide multiple languages for the log messages :zipper_mouth_face:

Reply:
Jetzt redest du!